src/main/resources/openapi/specs/api.kuflow.com/v1/openapi.yaml

```diff
@@ -331,13 +335,17 @@ paths:
   /tasks:
     get:
       summary: Find all accessible Taks
-      description: List all Tasks that have been created and the the credentials has access.
+      description: |
+        List all Tasks that have been created and the the credentials has access.
+
+        Available sort query values: id, createdDate, lastModifiedDate, claimedDate, completedDate, cancelledDate
```

Comment:
For date attributes we are using the the suffix “at” so in this case should be: createdAt, lastModifuedAt…

Reply:
In openapi, I mean